hoomd/md/pytest/test_dihedral.py

```diff
@@ -35,21 +35,21 @@
         0.0852,
     ),
     (
-        hoomd.md.dihedral.Harmonic,
+        hoomd.md.dihedral.Periodic,
         dict(),
         dict(k=3.0, d=-1, n=2, phi0=numpy.pi / 2),
         0,
         3,
     ),
     (
-        hoomd.md.dihedral.Harmonic,
+        hoomd.md.dihedral.Periodic,
         dict(),
         dict(k=10.0, d=1, n=1, phi0=numpy.pi / 4),
         5.0,
         5.0,
     ),
     (
-        hoomd.md.dihedral.Harmonic,
+        hoomd.md.dihedral.Periodic,
         dict(),
         dict(k=5.0, d=1, n=3, phi0=numpy.pi / 6),
         1.9411,
```

Comment:
The three modified test cases are just repeats of the cases above now. Let's remove them instead.